src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/property-info": "~2.8|3.0",
         "symfony/security": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/validator": "~3.1",
```

Comment:
I think the point is that the test suite tests both against the highest and lowest possible dependencies. Since 3.1 is the minimum boundary, even the  `--prefer-lowest` tests will run against 3.1 and not against 2.8, which might hide compatibility issues.